vscode-lean4/src/utils/leanpkg.ts

```diff
@@ -3,6 +3,7 @@ import { LocalStorageService} from './localStorage'
 import { findLeanPackageRoot, findLeanPackageVersionInfo } from './projectInfo';
 import * as fs from 'fs';
```

Comment:
You can now remove `import * as fs from 'fs';` -- notice in VS code how this line is "dimmed" - this is a que that the import is no longer needed.  I think the same is true in some other files you changed.

Reply:
Sure! good catch!